src/Controller/Controller_Actions.php

```diff
@@ -173,8 +179,20 @@ public function route_notices() {
 					]
 				);
 
-				$class = ( isset( $route['view_class'] ) ) ? $route['view_class'] : '';
-				$this->notices->add_notice( call_user_func( $route['view'], $route['action'], $route['action_text'] ), $class );
+				$message = call_user_func( $route['view'], $route['action'], $route['action_text'] );
+				/* Lets load the Install Core font message to Helper_Notice if the current page belongs to Gravity PDF or the plugin directory. */
+				if ( $is_gfpdf_page || $pagenow === 'plugins.php' ) {
+					$class = ( isset( $route['view_class'] ) ) ? $route['view_class'] : '';
+					$this->notices->add_notice( $message, $class );
+
+				} else {
+					/* Add  Install Core font message GFCommon if it doesn't. Remove the font styling to match with the generic add-on error message. */
+					if ( ! empty( \GFCommon::$errors ) ) {
+						$message = '<br>' . preg_replace( '/(<[^>]+) style=".*?"/i', '$1', $message );
```

Comment:
Remove this preg_replace. We can probably always wrap message in a `<p>` or `<div>` tag if the message displays inline with an existing message.